net.runelite.RuneLite.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "net.runelite.RuneLite",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "runelite",
+    "separate-locales": false,
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk10" 
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--env=JAVA_HOME=/app/jre",
+        "--persist=~"
```

Comment:
I am not sure if this is actually used. The RuneLite updater seems to be downloading all the game files every time.

Reply:
(You can make multiple `--persist` options, for example `--persist=.foo, --persist=.bar` will make the ~/.foo and ~/.bar files/directories persistent.)